jobs/settings/base.py

```diff
@@ -2,11 +2,11 @@
 from datetime import timedelta
 from distutils.util import strtobool
 
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
 SECRET_KEY = '@pzqp#x^+#(olu#wy(6=mi9&a8n+g&x#af#apn07@j=5oin=xb'
 
-DEBUG = strtobool(os.environ.get('APF_DEBUG', False))
+DEBUG = strtobool(os.environ.get('APF_DEBUG', 'False'))
```

Comment:
Quick note on this change, I noticed that has been introduced recently but it was failing the string to bool conversion for the boolean `False` so I changed the default value into a String

Reply:
Perfect! ty